app/views/oral_history_delivery_mailer/oral_history_delivery_email.html.erb

```diff
@@ -0,0 +1,11 @@
+<p>Dear <%= mailer.patron_name %>,</p>
+<p>Thanks for your recent visit to the <%= link_to "Science History Institute’s digital collections website", mailer.hostname %>. On <%=mailer.created_at.strftime("%m/%d/%Y")%>, you requested access to the files for <%= mailer.work_link %>. The files are ready for you to download.</p>
+<h2>Files for <%= mailer.work_link %>:</h2>
+<ul>
+  <% mailer.asset_links.each do |link| %>
+    <li><%= link_to link[:text], link[:url]%></li>
```

Comment:
We could give them the file type (PDF, MP3, FLAC), and the file size here too. Would that be nice of us? Or is this really just a placeholder until we do more review, and we don't care about polishing it more now?

Reply:
It certainly did occur to me. It's a quick change; I think I'll just add it in.